docs/api-guide.md

```diff
@@ -894,60 +944,59 @@ public class Sample {
 }
 ```
 
-The `begin()` API could throw `TransactionException` or `TransactionNotFoundException`.
-If you catch `TransactionException`, it indicates that the transaction has failed to begin due to transient or nontransient faults. You can try retrying the transaction, but you may not be able to begin the transaction due to nontransient faults.
-If you catch `TransactionNotFoundException`, it indicates that the transaction has failed to begin due to transient faults. You can retry the transaction.
+### `TransactionException` and `TransactionNotFoundException`
+
+The `begin()` API could throw `TransactionException` or `TransactionNotFoundException`:
+
+- If you catch `TransactionException`, this exception indicates that the transaction has failed to begin due to transient or non-transient faults. You can try retrying the transaction, but you may not be able to begin the transaction due to non-transient faults.
+- If you catch `TransactionNotFoundException`, this exception indicates that the transaction has failed to begin due to transient faults. In this case, you can retry the transaction.
+
```

Comment:
Should we mention how the `join()` API could also throw exceptions like we do in [`two-phase-commmit-transactions.md`](https://github.com/scalar-labs/scalardb/blob/f4360781940685d5c9bc86daa425fca870ae7f2f/docs/two-phase-commit-transactions.md?plain=1#L648C14-L648C14)?

> **Note**
> 
> **Reference:** The commit above is from PR #1082.

```suggestion

The `join()` API could also throw `TransactionException` or `TransactionNotFoundException`. You can handle these exceptions in the same way that you handle the exceptions for the `begin()` API.

```

Reply:
Actually, `DistributedTransactionManager.join()` only throws `TransactionNotFoundException`. Can you please mention this?

Reply:
Understood. 

I've added a sentence about that below this bulleted list. I've also updated the sentence in [two-phase-commit-transactions.md](https://github.com/scalar-labs/scalardb/blob/32a6191ff24f391175cf78217afc967086f80558/docs/two-phase-commit-transactions.md?plain=1#L654). PTAL!